### ISSUE TITLE
tldr-node can't render page with 2 line backtick quotes

### ISSUE THREAD

**Issue by lizhuoli1126** [tldr-pages/tldr-node-client#46]
From the official page lib, here are some pages with 2 or multi-line backtick quotes, for example:
https://github.com/tldr-pages/tldr/blob/master/pages/common/zpool.md

```markdown
- Show the history of all pool operations

`zpool histrory {{poolname}}`

- Create a mirrored pool.

`zpool create {{poolname}} mirror {{disk1}} {{disk2}}`
`zpool create {{poolname}} mirror {{disk1}} {{disk2}} mirror {{disk3}} {{disk4}}`
```

which is displayed in the [web client](http://www.ostera.io/tldr.jsx/) and [C++ client](https://github.com/tldr-pages/tldr-cpp-client), but cause blank in this node client with version *1.3.0*.

Here is the output:

```
➜  ~ tldr zpool

  zpool
  Manage ZFS pools

  - Show the configuration and status of all ZFS zpools
    zpool status [poolname]

  - Check a ZFS pool for errors (verifies the checksum of EVERY block). Very CPU and disk intensive.
    zpool scrub poolname

  - List zpools available for import
    zpool import

  - Export a zpool (unmount all filesystems)
    zpool export poolname

  - Show the history of all pool operations
    zpool histrory poolname


➜ ~ 
```

I think this is a legal grammar to let user know a little more detailed usage for current command. So could you fix this problem?

**Comment by lizhuoli1126** [comment]
OK..Maybe we should fix that page but not `tldr` ? And also the web and C++ client...

I read [CONTRIBUTING](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) and find there is not a grammar about `one description one command line`. Hope for someone to tell a final choice.

**Comment by igorshubovych** [comment]
According to Contributing Guidelines there should be single code example per single code description. Check Markdown format section.

@rprieto we should say it in Contributing Guidelines more directly.

There is hanging pull request about it #27 about it. We decided not to go this way.

**Comment by igorshubovych** [comment]
I am closing this for now. We can reopen this if we change guidelines.
